code/lib/preview-api/src/modules/store/csf/prepareStory.ts

```diff
@@ -98,26 +99,57 @@ export function prepareStory<TRenderer extends Renderer>(
     storyAnnotations?.render ||
     componentAnnotations.render ||
     projectAnnotations.render;
-  if (!render) throw new Error(`No render function available for storyId '${id}'`);
 
   const decoratedStoryFn = applyHooks<TRenderer>(applyDecorators)(undecoratedStoryFn, decorators);
   const unboundStoryFn = (context: StoryContext<TRenderer>) => decoratedStoryFn(context);
 
   const playFunction = storyAnnotations?.play ?? componentAnnotations?.play;
 
+  const mountUsed = mountDestructured(playFunction);
+
+  if (!render && !mountUsed) {
+    throw new Error(`No render function available for storyId '${id}'`);
+  }
+
+  let { tags } = partialAnnotations;
+
+  if (mountUsed) {
+    // Play is not supported in docs yet, and when mount is used, the mounting is happening in play itself.
+    tags = tags.filter((tag) => tag !== 'autodocs');
+  }
+
+  const defaultMount = (context: StoryContext) => {
+    return async () => {
+      await context.renderToCanvas();
+      return context.canvas;
+    };
+  };
```

Comment:
We could consider adding this as a default project-level annotation (via the default preset) rather than defining it in here? I don't know if we have a pattern here in how we do it.

Reply:
As far as I know, we don't really have a default project annotation in a default preset. But even if we had, I think portable stories is ignoring that. So this seems better to me.